byterun/io.c

```diff
@@ -118,25 +154,45 @@ CAMLexport void caml_close_channel(struct channel *channel)
   unlink_channel(channel);
   caml_stat_free(channel->name);
   caml_stat_free(channel);
+  return;
+}
+
+static io_result caml_lseek_end(int fd, file_offset* end)
+{
+  caml_enter_blocking_section();
+  *end = lseek(fd, 0, SEEK_END);
+  caml_leave_blocking_section_nosig();
+  if (*end == -1) return errno;
+  return 0;
+}
+
+static io_result caml_lseek_set(int fd, file_offset offset)
```

Comment:
```suggestion
static io_result lseek_set(int fd, file_offset offset)
```

It seems like the convention is to not use the `caml_` prefix for static functions.